helpers/class.ExtensionHelper.php

```diff
@@ -136,21 +139,20 @@ public static function mustBeEnabled(common_ext_Extension $extension)
         }
         return false;
     }
-    
+
     /**
      * Check if extension requirements are met
      * Always returns true, but throws exception on error
      *
      * @param common_ext_Extension $extension
-     * @throws common_ext_MissingExtensionException
-     * @throws common_ext_OutdatedVersionException
-     * @throws common_exception_Error
      * @return boolean
+     * @throws ManifestNotFoundException
+     * @throws common_ext_MissingExtensionException
      */
     public static function checkRequiredExtensions(common_ext_Extension $extension)
     {
         $extensionManager = common_ext_ExtensionsManager::singleton();
-        // read direct dependencies from manifest, do not check recursivly
+        // read direct dependencies from manifest, do not check recursively
         foreach ($extension->getManifest()->getDependencies() as $requiredExt => $requiredVersion) {
             $installedVersion = $extensionManager->getInstalledVersion($requiredExt);
             if (is_null($installedVersion)) {
```

Comment:
I think it would be better to change these two lines to `if (!$extensionManager->isInstalled($requiredExt)) {...}` to make it clear what do we check here.